### PR TITLE
adding litmus tutorial

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Adding a tutorial on how to use Litmus + Keptn to evaluate the resilience of microservices

### CODE REVIEW

**Comment by jetzlstorfer** [comment]
Preview: https://6059e18d8cce000007b0bc0f--keptn-tutorials.netlify.app/

File: site/tutorials/keptn-litmus-08.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+authors: Jürgen Etzlstorfer
+Feedback Link: https://github.com/keptn/tutorials/tree/master/site/tutorials
+
+
+# Resilience evaluation with LitmusChaos, Prometheus, and Keptn
+
+## Welcome
+Duration: 2:00 
+
+In this tutorial we'll set up a demo application and have it undergo some chaos in combination with load testing. We will then use Keptn quality gates to evaluate the resilience of the application based on SLO-driven quality gates.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:15]: In this tutorial, (< comma)

File: site/tutorials/keptn-litmus-08.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+You'll find a time estimate until the end of this tutorial in the right top corner of your screen - this should give you guidance how much time is needed for each step.
+
+
+In this tutorial, we are going to install Keptn on a Kubernetes cluster.
+
+The full setup that we are going to deploy is sketched in the following image.
+![demo setup](./assets/keptn-litmus/demo-workflow.png)
+
+If you are interested, please have a look at this presentation from Litmus and Keptn maintainers presenting the intial integration.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:31]: the **initial** integration.

File: site/tutorials/keptn-litmus-08.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "direct"
+            - name: "test"
+              properties:
+                teststrategy: "performance"
+            - name: "evaluation"
+```
+
+In the `shipyard.yaml` shown above, we define a single stage called *chaos* with a single sequence called *delivery*. In this sequence, a *deployment*, *test*, and *evaluation*  task is defined (along with some properties). With this, Keptn sets up the environment and makes sure, that tests are triggered after each deployment, and the tests are then evaluted by Keptn quality gates. As we do not have a subsequent stage, we do not need an *approval* or *release* task.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:176]: ... then **evaluated** by ...

File: site/tutorials/keptn-litmus-08.md [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./jmeter/load.jmx --resourceUri=jmeter/load.jmx
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./jmeter/jmeter.conf.yaml --resourceUri=jmeter/jmeter.conf.yaml
+    ```
+
+Now each time Keptn triggers the test execution, the JMeter service will pick up both files and execute the tests.
+
+## Configure Quality Gate
+Duration: 2:00
+
+We have not yet added our quality gate, i.e., the evalution of several SLOs done by Keptn. Let's do this now!
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:203]: ... the **evaluation** of ...

File: site/tutorials/keptn-litmus-08.md [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+
+We are now ready to kick off a new deployment of our test application with Keptn and have it deployed, tested, and evaluated.
+
+1. Let us now trigger the deployment, tests, and evaluation of our demo application.
+
+    ```
+    keptn trigger delivery --project=litmus --service=helloservice --image=jetzlstorfer/hello-server:v0.1.1
+    ```
+
+1. Let's have a look in the Keptn's bridge what is actually going on. We can use this helper command to retrieve the URL of our Keptn's bridge.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:270]: Please change: Keptn`s bridge to --> Keptn Bridge.

(Some time ago, I got feedback from a technical writer that the `s should not be used.)

File: site/tutorials/keptn-litmus-08.md [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+    keptn configure bridge --output
+    ```
+
+    ![](./assets/keptn-litmus/litmus-first-run-started.png)
+
+1. We can see that the evaluation failed, but why is that?
+
+    ![](./assets/keptn-litmus/litmus-first-run.png)
+
+1. Lets take a look at the evalution - lick on the *chart* icon in the red evaluation tile.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:288]: **Let's** take a look at the **evaluation** ...

File: site/tutorials/keptn-litmus-08.md [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+
+    ![](./assets/keptn-litmus/litmus-first-evaluation.png)
+
+    We can see that the evaluation failed because both the `probe_duration_ms` as well as the `probe_success_percentage` SLOs did not meet their criteria.
+    Considering the fact that our chaos experiment did delete the pod of our application, we might want to increase the number of replicas that are running to make our applcation more resilient. Let's do this in the next step.
```
Comment (johannes-b) [inline site/tutorials/keptn-litmus-08.md:293]: ... **application** ...

**Comment by jetzlstorfer** [comment]
Thanks for finding the typos @johannes-b

**Review by johannes-b — APPROVED** [review]
lgtm

**Comment by jetzlstorfer** [comment]
thanks - will merge as soon as the release branch of the integration is ready

**Pull request merged**